packages/app-store/huddle01video/lib/VideoApiAdapter.ts

```diff
@@ -1,46 +1,143 @@
-import z from "zod";
-
-import { handleErrorsJson } from "@calcom/lib/errors";
-import { randomString } from "@calcom/lib/random";
+import logger from "@calcom/lib/logger";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+import type { CredentialPayload } from "@calcom/types/Credential";
 import type { PartialReference } from "@calcom/types/EventManager";
-import type { VideoApiAdapter, VideoCallData } from "@calcom/types/VideoApiAdapter";
+import type { VideoApiAdapter } from "@calcom/types/VideoApiAdapter";
+
+import { getHuddle01APIKey, getHuddle01Credential } from "../utils/storage";
+
+const API_END_POINT = "https://platform-api-darshan.huddle01.workers.dev/api/v2/calendar";
```

Comment:
change url before opening PR

also would suggest using one seperate file to store all constants like endpoint urls etc